src/Core/src/Core/IBoxView.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	/// <summary>
+	/// BoxView renders a simple rectangle of a specified width, height, and color. 
+	/// </summary>
+	public interface IBoxView : IView
+	{
+		/// <summary>
+		/// Gets the color which will fill the rectangle.
+		/// </summary>
+		Color Color { get; }
```

Comment:
Should this be a Paint ? or a Brush ?

Reply:
Also, there is no need for a special `Color` property, at least in core. `BackgroundColor/Brush` would do. The Controls can map to that property to stay backward compatible.

Reply:
created a ussye ti follow up on this https://github.com/dotnet/maui/issues/2365